### PR TITLE
Plotly.js renderer

### PR DESCRIPTION
**Pull request by rgbkrk**
This introduces a Plotly transform for the `application/json+plotly.v1` mimetype.

screen shot 2016-08-30 at 7 19 33 pm

/cc @jackparmer @chriddyp @captainsafia

### CODE REVIEW

**Comment by jdfreder** [comment]
slick

File: src/notebook/components/transforms/plotly.js [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+    Plotly.newPlot(this.el, payload.data, payload.layout);
+  }
+
+  shouldComponentUpdate() {
+    return false;
+  }
+
+  render() {
+    return (
+      <div ref={(el) => this.el = el} /> // eslint-disable-line
```
Comment (chriddyp) [inline src/notebook/components/transforms/plotly.js]: I'm not sure how the output cells work w.r.t. sizing, but we might need to set `height` here if `layout.height && !layout.autosize`. Here's an example figure:

```
data = [{x: [1, 2], y: [3, 1]}]
layout = {height: 1600}
```

Comment (rgbkrk) [reply src/notebook/components/transforms/plotly.js]: The cell allows the output to be any size it wants to be. We can set it explicitly here like you said.

``` js
render() {
  const { data, layout } = this.props.data.toJS();
  const style = {}
  if(layout.height && !layout.autosize) {
    style.height = layout.height;
  }
  return (
    <div style={style} ref={(el) => this.el = el} /> // eslint-disable-line
  )
}
```

**Comment by captainsafia** [comment]
Merged! 🎉 

Slight coverage drop but we can address that in subsequent PRs.

**Pull request merged**